packages/functional-tests/playwright.config.ts

```diff
@@ -1,4 +1,5 @@
-import { PlaywrightTestConfig, Project } from '@playwright/test';
+import { PlaywrightTestConfig, defineConfig } from '@playwright/test';
+import type { Project } from '@playwright/test';
 import * as path from 'path';
 import { TargetNames } from './lib/targets';
 import { TestOptions, WorkerOptions } from './lib/fixtures/standard';
```

Comment:
nit: we could organize here
```suggestion
import type { Project } from '@playwright/test';
import { PlaywrightTestConfig, defineConfig } from '@playwright/test';
import * as path from 'path';
import { TestOptions, WorkerOptions } from './lib/fixtures/standard';
import { TargetNames } from './lib/targets';
```